website/sidebars.json

```diff
@@ -88,16 +88,23 @@
       },
       "cita/store",
       {
-        "label": "Hacking",
+        "label": "Advanced-use",
         "type": "subcategory",
         "ids": [
-          "cita/hacking/compile-cita",
-          "cita/hacking/log",
-          "cita/hacking/auto-exec",
-          "cita/hacking/chain-gov",
-          "cita/hacking/contracts/solidity",
-          "cita/hacking/contracts/rust",
-          "cita/hacking/contracts/batch-tx"
+          "cita/advanced-use/contracts/solidity",
+          "cita/advanced-use/contracts/rust",
+          "cita/advanced-use/auto-exec",
+          "cita/advanced-use/chain-gov",
+          "cita/advanced-use/contracts/batch-tx"
+        ]
+      },
+      {
+        "label": "Ops",
```

Comment:
Is it better to use Chinese?

Reply:
Check the `TODO` part. 

https://github.com/cryptape/citahub-docs/blob/master/website/sidebars.json

Reply:
Got it.

Reply:
The chinese version has to be uploaded using crowdin.